build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:7.1.3'
```

Comment:
This is all, frankly, an abomination. the v34 compile sdk thinks gradle 7.x is too old, but any newer version of gradle requires an opencv upgrade to >= 4.9, which (due to api breakage) requires rewriting a far chunk of the app.

This'll all come crashing down within the next year, *or* I'll figure out a way to get it sorted out, *or* I'll pull the app from the play store.